src/Mod/Sketcher/Gui/DrawSketchHandlerSplitting.h

```diff
@@ -90,7 +93,10 @@ class DrawSketchHandlerSplitting: public DrawSketchHandler
             const Part::Geometry *geom = sketchgui->getSketchObject()->getGeometry(GeoId);
             if (geom->getTypeId() == Part::GeomLineSegment::getClassTypeId()
                 || geom->getTypeId() == Part::GeomCircle::getClassTypeId()
-                || geom->getTypeId() == Part::GeomArcOfCircle::getClassTypeId()) {
+                || geom->getTypeId() == Part::GeomEllipse::getClassTypeId()
+                || geom->getTypeId() == Part::GeomArcOfCircle::getClassTypeId()

```

Comment:
Isn't GeomArcOfCircle a redundant checking in view of isDerivedFrom(GeomArcOfConic)?

Reply:
I was thinking the same. Just procrastinated deciding whether to make a new commit or amend an old one.

Reply:
Changes pushed.

Reply:
This kind, I generally make a new commit, interactive rebase to move it up after the commit that needs correction, and squash it together with it. Then I force-push the branch. It is the cleanest history line.